<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="outTable">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>231</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="RecBlinkLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Recorded Blinks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>STATUS</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="seedEntry">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DEADBEEF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seeInputButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>^ See inputs</string>
    </property>
   </widget>
   <widget class="QLabel" name="TotalTimeLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TOTAL REMAINING:</string>
    </property>
   </widget>
   <widget class="QLabel" name="localTimeLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Next blink:</string>
    </property>
   </widget>
   <widget class="QLabel" name="delayLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU DELAY</string>
    </property>
   </widget>
   <widget class="QLabel" name="exitTimeLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT ON:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="arbTargetBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>450</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="increaseBlinksButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>470</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add 10 more blinks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>test targetSelection</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuGithub">
    <property name="title">
     <string>GitHub</string>
    </property>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
   <addaction name="menuGithub"/>
   <addaction name="menuExit"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
